docker-compose.yml

```diff
@@ -11,6 +11,7 @@ services:
     # generate /ssl/dhparam.pem with openssl dhparam -out dhparam.pem 4096
       - ./nginx/locations:/etc/nginx/locations
       - ./nginx/ssl:/ssl
+      - ./nginx/conf/templates:/etc/nginx/templates
```

Comment:
What does this help with? Are you overriding something in the `default.conf` or adding a new `.conf`?

Reply:
e.g., 
```
nginx/conf/templates/default.conf.template
diff --git a/nginx/conf/templates/default.conf.template b/nginx/conf/templates/default.conf.template
index a53258b..0054deb 100644
--- a/nginx/conf/templates/default.conf.template
+++ b/nginx/conf/templates/default.conf.template
@@ -31,7 +31,7 @@ server {

     error_log  syslog:server=syslog:5140 notice;
     access_log syslog:server=syslog:5140 timed_combined;
-    listen 443 ssl;
+    listen 8443 ssl;
     server_name ${NGINX_SERVER_NAMES};
     #limit_conn conn_per_ip 50;
     #limit_req zone=req_per_ip burst=100 nodelay;
```

it was to match the outside port because one of the application was misbehaving, might have been solved by the proxy port change below that was done later

Reply:
Right, my issue here is that this will always override the `default.conf.template`; so to deploy a new version you need to `git pull` && `docker compose pull`. But maybe one should always do that, to get new things in the compose file..
Thoughts?